zerver/views/webhooks/github.py

```diff
@@ -14,17 +14,20 @@
 ZULIP_TEST_REPO_ID = 6893087
 
 def is_test_repository(repository):
+    # type: (Dict[str, Any]) -> bool
     return repository['name'] == ZULIP_TEST_REPO_NAME and repository['id'] == ZULIP_TEST_REPO_ID
 
 class UnknownEventType(Exception):
     pass
 
 
 def github_generic_subject(noun, topic_focus, blob):
+    # type: (text_type, text_type, Dict[text_type, Any]) -> bool
```

Comment:
The return type should `text_type`, not `bool`. And import `text_type` from `six`.